tensilelite/Tensile/Source/lib/source/AMDGPU.cpp

```diff
@@ -49,6 +56,27 @@ namespace TensileLite
 
     TENSILE_API AMDGPU::~AMDGPU() = default;
 
+    TENSILE_API bool AMDGPU::isStandardCU() const
+    {
+        // return the result if we already tested it.
+        if(isStandardCUs != -1)
+            return (isStandardCUs == 1);
+
+        // assume current device is a standard cu device.
+        isStandardCUs = 1;
+
```

Comment:
That's why I declare it as a mutable var: I'd like to save the result to this var in the first call. (so that we don't need to do the following testing every time.)